### PR TITLE
Fix/31237/new button app list

### PR DESCRIPTION
**Pull request by vanpertsch**
Capture d’écran_2022-02-28_14-35-58

https://github.com/nextcloud/server/issues/31237#issuecomment-1048756457

### CODE REVIEW

File: apps/settings/css/settings.scss [hunk 1]
```diff
@@ -1040,15 +1040,15 @@ span.version {
 		background-color: var(--color-main-background);
 		position: fixed;
 		z-index: 1;
-		height: $header-height;
+		height: $header-height + 10px;
```
Comment (skjnldsv) [inline apps/settings/css/settings.scss:1043]: We usually try to stay away from random numeric values like those :)
I would adapt the header instead to fit everything properly :thinking:

**Comment by jancborchardt** [comment]
Right now all buttons are primary, would be nicer if the "Update all" is the only primary one, and the individual app update buttons are secondary.

File: apps/settings/css/settings.scss (hunk 1 above)
Comment (vanpertsch) [reply apps/settings/css/settings.scss:1043]: @skjnldsv i understand what you mean with the numeric values. But i am not sure what you mean by "_adapt the header instead_".

**Comment by vanpertsch** [comment]
@jancborchardt the _individual app update buttons_ are not yet vue-components. (they are not buttons, but input fields)

**Review by jancborchardt — APPROVED** [review]
Thanks for clarifying @vanpertsch, looks good design-wise then :)

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Capture d’écran_2022-02-28_14-35-58

**Pull request merged**